# Vulnerability Report for getwilds/scanpy:latest

Report generated on Tue Apr  1 06:21:13 UTC 2025

    ...Pulling
    âœ— Pull failed
[31mERROR  [0m Status: failed to get image getwilds/scanpy:latest: failed to pull image getwilds/scanpy:latest: GET https://index.docker.io/v2/getwilds/scanpy/manifests/latest: TOOMANYREQUESTS: You have reached your pull rate limit as 'tfirmanfh': dckr_jti_qkSPr2vZP2ieCB5xAt8gVsB3uLY=. You may increase the limit by upgrading. https://www.docker.com/increase-rate-limit, Code: 1 
Error generating all vulnerabilities report. Please try again later.
